proposed/http-client/http-client.md

```diff
@@ -62,8 +61,8 @@ interface ClientInterface
      * Sends a PSR-7 request and returns a PSR-7 response. 
      * 
      * Every technically correct HTTP response MUST be returned as is, even if it represents a HTTP 
-     * error response or a redirect instruction. The only exception is 1xx responses which MUST be 
-     * assembled in the HTTP client. 
+     * error response or a redirect instruction. The only exception are 1xx responses which MUST be 
```

Comment:
Hm, Im not sure you are correct here. The verb ("be") i referring to "the only exception" which is singular.

Please, any native speaker, could you correct this?

Reply:
`The only exceptions are 1xx responses, which MUST be `

Reply:
`exceptions are` or `exception is`. You could use either of those two this case.

Reply:
Thank you.